src/thor/astar.cc

```diff
@@ -237,12 +237,13 @@ void AStarPathAlgorithm::ExpandForward(GraphReader& graphreader,
 }
 
 // Calculate best path. This method is single mode, not time-dependent.
-std::vector<PathInfo>
+std::vector<std::vector<PathInfo>>
```

Comment:
the path algorithms all are allowed to return more than one path now. none of them do, but the api allows for it